app/webpacker/components/Tickets/TicketCommentCreate.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Button, Modal } from 'semantic-ui-react';
+import { useMutation } from '@tanstack/react-query';
+import MarkdownEditor from '../wca/FormBuilder/input/MarkdownEditor';
+import useInputState from '../../lib/hooks/useInputState';
+import createComment from './api/createComment';
+import Loading from '../Requests/Loading';
+import Errored from '../Requests/Errored';
+
+export default function TicketCommentCreate({
+  open, onClose, ticketId, currentStakeholder, refetchComments,
+}) {
+  const [comment, setComment] = useInputState();
+
+  const {
+    mutate: doCreateComment,
```

Comment:
Instead of the `do...` prefix, maybe call it `createCommentMutation`? That also avoids the naming collision with the API function.